runtime/autoload/man.vim

```diff
@@ -1,6 +1,11 @@
-let s:man_tag_depth = 0
+" Ensure Vim is not recursively invoked (man-db does this)
+" More info here http://comments.gmane.org/gmane.editors.vim.devel/29085
+let s:man_cmd = 'unset MANPAGER; man 2>/dev/null '
```

Comment:
Note that `unset MANPAGER; man 2>/dev/null` will fail for `set shell=fish` (which Vim claims to support in some capacity—not sure if that extends to default plugins). Fish syntax would be `set -e MANPAGER; man ^/dev/null`.